model-optimizer/mo/back/offline_transformations.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+if __name__ == "__main__":
```

Comment:
please, make sure that this file has executable bit set on *NIX machines.

Reply:
Decided to remove #!/usr/bin/env python3 from all files as they are useless and in all cases we execute this scripts using specific python version.